plugins/modules/keycloak_identity_provider.py

```diff
@@ -124,6 +124,15 @@
       - providerId
     type: str
 
+  hide_on_login:
+    description:
+      - If hidden, login with this provider is possible only if requested explicitly, for example using the C(kc_idp_hint)
```

Comment:
I think you forgot one line from the original parameter:
```suggestion
      - If hidden, login with this provider is possible only if requested explicitly, for example using the C(kc_idp_hint)
        parameter.
```

Reply:
BTW, what/where is this `kc_idp_hint` parameter? It does not seem to be in the module whatsoever. Should it be? Should we use `O()` instead of `C()`?

Reply:
Hi,
The kc_idp_hint is a query parameter that a client (i.e., a client application) can use to suggest a specific identity provider during user authentication.
For more information:
https://www.keycloak.org/docs/latest/server_admin/index.html#_client_suggested_idp